static/sass/_pattern_blog-suru.scss

```diff
@@ -0,0 +1,31 @@
+@mixin blog-p-strips-suru {
+  $suru-bg-color: rgba(17, 17, 17, 0.65);
+
+  .p-strip--blog-suru {
```

Comment:
Can we use an existing strip? For example [topped](https://vanillaframework.io/docs/patterns/strip#topped-suru-strip)?